opencog/openpsi/mock-interaction-rules.scm

```diff
@@ -0,0 +1,57 @@
+;
+; mock-interaction-rules.scm
+;
```

Comment:
This entire file belongs in `ros-behavior-scripting` or something like that, but certainly not in this directory.  

I would prefer to have it in `ros-behavior-scripting` so that it can be a part of the self model (so that if you ask the robot "what are you doing?" it could answer appropriately)